composer.json

```
@@ -1,36 +1,25 @@
 {
   "name": "psr/log-util",
   "description": "Utilities for the psr/log package that backs the PSR-3 specification.",
-  "minimum-stability": "dev",
   "license": "MIT",
   "authors": [
     {
       "name": "PHP-FIG",
       "homepage": "https://www.php-fig.org/",
       "role": "Organisation"
-    },
-    {
-      "name": "Anton Ukhanev",
-      "email": "xedin.unknown@gmail.com",
-      "role": "Developer"
     }
   ],
   "require": {
     "php": "^5.3 | ^7.0 | ^8.0",
     "psr/log": "^1.1.4"
```

I would lower this as much as possible
```suggestion
    "psr/log": "^1.1.0"
```

---

There is not significant changes ([diff](https://github.com/php-fig/log/compare/1.1.0...1.1.4)), we can accept `1.1.0` to avoid any issue for users.

---

`psr/log: 1.1.1` minimum is required for PHPUnit compatibility.
https://github.com/php-fig/log/commit/c83175653e5ad0b205011537b6c534eaa767f52d

Since composer dependency resolution will always go for the latest version, I think this version constraint is fine.

---

I was worrying about the ton of CI processes with `--prefer-lowest`; `^1.1.1` seems fine to me:
```suggestion
    "psr/log": "^1.1.1"
```